compose-imageviewer/webApp/src/wasmJsMain/kotlin/Main.wasm.kt

```diff
@@ -5,14 +5,14 @@ import example.imageviewer.*
 import example.imageviewer.model.*
 import org.jetbrains.compose.resources.ExperimentalResourceApi
 import org.jetbrains.compose.resources.configureWebResources
-import org.jetbrains.compose.resources.urlResource
+//import org.jetbrains.compose.resources.urlResource
 import kotlin.wasm.unsafe.*
 
 @OptIn(ExperimentalComposeUiApi::class, ExperimentalResourceApi::class)
 fun main() {
     configureWebResources {
         // same as default - this is not necessary to add here. It's here to show this feature
-        setResourceFactory { urlResource("./$it") }
+        //setResourceFactory { urlResource("./$it") }
```

Comment:
```suggestion
        resourcePathMapping { path -> "./$path" }
```

Reply:
But again - it doesn't really make any difference because the default is the same. It's just to show the feature.

Reply:
@eymar pushed this fix